edx_event_bus_kafka/internal/producer.py

```diff
@@ -112,7 +112,7 @@ def descend_avro_schema(serializer_schema: dict, field_path: List[str]) -> dict:
 
             matching = [field for field in field_list if field['name'] == field_name]
             subschema = matching[0]
-        except BaseException as e:
+        except Exception as e:  # pylint: disable=broad-except
```

Comment:
Interestingly, pylint says this is a "useless suppression" -- I think because the exception is reraised.